server/common/ResponseBuilder.java

```diff
@@ -71,7 +71,7 @@ public static StatusRuntimeException exception(Throwable e) {
             return (StatusRuntimeException) e;
         } else {
             return Status.INTERNAL.withDescription(
-                    e.getMessage() + " Please check server logs for the stack trace."
+                    e.getMessage() + "\n\nPlease check server logs for the stack trace."
```

Comment:
Previously, the resulting error would be something like:
```
[GQL03] TypeQL Error: There is a syntax error at line 1:
match $x sb thing;
         ^
no viable alternative at input 'match $x sb' Please check server logs for the stack trace.
```
Now, the message will be:
```
[GQL03] TypeQL Error: There is a syntax error at line 1:
match $x sb thing;
         ^
no viable alternative at input 'match $x sb'

Please check server logs for the stack trace.
```
This ensures that the actual error is cleanly separated from the instruction.